src/revlanguage/main.cpp

```diff
@@ -53,14 +77,14 @@ variables_map parse_cmd_line(int argc, char* argv[])
     // RevBayes doesn't use a global verbose_logging flag.
     // ("verbose,V",value<int>()->implicit_value(1),"Log extra information for debugging.")
 
-	("batch,b","Run in batch mode.")
+	("batch,b","Run in batch mode (i.e. RevBayes will exit if it encounters an error).")
     ("jupyter,j","Run in jupyter mode.")
     // multitoken means that `--args a1 a2 a3` works the same as `--args a1 --args a2 --args a3`
-    ("args",value<std::vector<std::string> >()->multitoken(),"Command line arguments to initialize RevBayes variables.")
+    ("args",value<std::vector<std::string> >()->multitoken(),"Supply command-line arguments to RevBayes. These can be accessed from within the program using the 'args' vector. See ?args for details.")
+    // composing means that --file can occur multiple times
+    ("file",value<std::vector<std::string> >()->composing(),"Source one or more files.")
     // multitoken means that `--args a1 a2 a3` works the same as `--args a1 --args a2 --args a3`
-    ("cmd",value<std::vector<std::string> >()->multitoken(),"Script and command line arguments to initialize RevBayes variables.")
-	// composing means that --file can occur multiple times
-    ("file",value<std::vector<std::string> >()->composing(),"File(s) to source.")
+    ("cmd",value<std::vector<std::string> >()->multitoken(),"Source a file and supply command-line arguments in batch mode. Cannot be combined with the --args flag. If followed by the --file flag, the script sourced via --file is executed first.")
```

Comment:
The script with "command" should be executed first, I think.  So we should really fix this bug rather than describe it.

Reply:
I pushed a fix for that.